src/main/java/Lindi.java

```diff
@@ -0,0 +1,227 @@
+                if (taskList.size() == 0) {
+                    System.out.println("There are no tasks for you today. Enjoy :)");
+                }
+            }
+
+            void executeCreateTask(String s) {
+                Task task;
+                try{
+                    task = Task.create(s);
+                } catch (CreateTaskException e) {
+                    System.out.println(e.getMessage());
+                    return;
+                }
+                taskList.add(task);
+                System.out.println("Got it. I've added this task:\n\t" + task +
+                                   "\nNow you have " + taskList.size() + " tasks in the list.");
+                saveToFile();
+            }
+
+            /**
+             * Prints a goodbye message and Exits the program (return code 0)
+             */
+            void goodByeAndExit() {
+                System.out.println("Bye. Hope to see you again soon!");
+                printSeparator();
+                System.exit(0); // exit with code 0, terminates program
+            }
+
+            void executeModifyTask(String s) {
+                String[] sTokens = s.split(" ", 3); // expect exactly 2 substrings for a valid command input
+                if (sTokens.length == 1) {
+                    System.out.println("Uh oh ! You have to give me the index of the task you want to modify :)");
+                    return;
+                } else if (sTokens.length == 3) {
+                    System.out.println("Umm... is it that hard to give me JUST the index? :(");
+                    return;
+                }
+
+                try {
+                    int listIndex = Integer.parseInt(sTokens[1]);
+                    if (listIndex < 1 || listIndex > taskList.size()) {
+                        System.out.println("Sorry, I can't find that task. Please enter a valid index\n" +
+                                "You can see the tasks list by inputting \"list\"");
+                        return;
+                    }
+                    Task task = taskList.get(listIndex - 1);
+                    switch (sTokens[0]) {
+                        case "mark":
+                            task.markDone();
+                            System.out.println("Nice! I've marked this task as done:\n\t" + task);
+                            break;
+                        case "unmark":
+                            task.unmarkDone();
+                            System.out.println("Nice! I've marked this task as not done yet:\n\t" + task);
+                            break;
+                        case "delete":
+                            taskList.remove(listIndex - 1);
+                            System.out.println("Okay. I've deleted this task:\n\t" + task +
+                                    "\nNow you have " + taskList.size() + " tasks in the list.");
+                            break;
+                        default:
```

Comment:
Should the case clauses have indentations? I noticed the same issue in several other places too.